### DISCUSSION TITLE
Uploaded Files - Public when article is Private and not Shared

### DISCUSSION THREAD

**Discussion by Leonardo-Alcantar** [GitbookIO/community#590] [0. General bugs and questions] [answered]
Hello - I'm trying out GitBook for our organization. Everything is great so far. However, I uploaded a PDF file to one of our pages, and out of curiosity, I opened it to see how it opens - I then copied the URL and had someone else try to open it (the person doesn't have a GitBook account) - The PDF file opened without asking for a login. However, the page itself requires login, and it's not shared. I'm also the only person with an account. 

I don't see an option to make our files private unless otherwise shared. Is this by design, or is it a bug? We have some confidential internal PDFs we want to upload and share, but they can't be accessible on the web. According to some of the documentation I looked at, the sharing is based on the page, so if our page is private, then the uploaded PDF should be as well, correct?

Thank you

**Comment by aikoii** [comment, marked as answer]
👋 @Leonardo-Alcantar, unfortunately, embedded PDFs on your pages don't automatically inherit the doc's access restrictions. For sensitive documents, consider hosting them in secure cloud storage and linking to them from your GitBook pages. This way, you ensure security, although the PDFs won’t be embedded directly.

Remember, if a space is set to restrict access to your teammates only or through Visitor Authentication, only authorized users can view the space itself.
